packages/kotti-ui/source/kotti-banner/index.ts

```diff
@@ -12,8 +12,14 @@ export const KtBanner = attachMeta(makeInstallable(KtBannerVue), {
 			'https://www.figma.com/file/0yFVivSWXgFf2ddEF92zkf/Kotti-Design-System?node-id=1439%3A5',
 	},
 	slots: {
-		expand: { description: null, scope: null },
-		glyph: { description: null, scope: null },
+		expand: {
+			description: 'Content when Banner is expended',
+			scope: null,
+		},
+		glyph: {
+			description: 'Use if you want to showw something else than a yoco icon',
```

Comment:
1. Typo in `showw`
2. Can you check if we even use this? This might be removeable

Reply:
It's not used. Will remove since I'm not a fan